### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by alexlanz**

### CODE REVIEW

File: databrowser/src/components/hero/HeroSubTitle.vue [hunk 1]
```diff
@@ -1,3 +1,4 @@
+†
```
Comment (mmalfertheiner) [inline databrowser/src/components/hero/HeroSubTitle.vue]: I think this should be removed

File: databrowser/src/pages/AboutPage.vue [hunk 2]
```diff
@@ -1,3 +1,15 @@
 <template>
-  <h1>About Page</h1>
+  <AppLayout>
+    <ContentArea>About Page</ContentArea>
+  </AppLayout>
 </template>
+
+<script>
```
Comment (gappc) [inline databrowser/src/pages/AboutPage.vue:7]: I would set the `lang="ts"` info to improve tooling support (e.g. by VSCode `volar` extension).

At the moment, `volar` shows me this message:

```
Virtual script not found, may missing <script lang="ts"> / "allowJs": true / jsconfig.json.volar
```

File: databrowser/src/pages/HomePage.vue [hunk 3]
```diff
@@ -1,120 +1,121 @@
...
+          <router-link class="bg-red-100" to="/dataset/xyz"
+            >Not existing list route
+          </router-link>
+        </li>
+      </CardGrid>
+    </ContentArea>
+  </AppLayout>
 </template>
+
 <script>
```
Comment (gappc) [inline databrowser/src/pages/HomePage.vue:113]: I would set the `lang="ts"` info to improve tooling support (e.g. by VSCode `volar` extension).

At the moment, `volar` shows me this message:

```
Virtual script not found, may missing <script lang="ts"> / "allowJs": true / jsconfig.json.volar
```

File: databrowser/src/pages/ImprintPage.vue [hunk 4]
```diff
@@ -1,3 +1,15 @@
 <template>
-  <div>Imprint Page</div>
+  <AppLayout>
+    <ContentArea>Imprint Page</ContentArea>
+  </AppLayout>
 </template>
+
+<script>
```
Comment (gappc) [inline databrowser/src/pages/ImprintPage.vue:7]: I would set the `lang="ts"` info to improve tooling support (e.g. by VSCode `volar` extension).

At the moment, `volar` shows me this message:

```
Virtual script not found, may missing <script lang="ts"> / "allowJs": true / jsconfig.json.volar
```

File: databrowser/src/pages/LegalPage.vue [hunk 5]
```diff
@@ -1,3 +1,15 @@
 <template>
-  <div>Legal Page</div>
+  <AppLayout>
+    <ContentArea>Legal Page</ContentArea>
+  </AppLayout>
 </template>
+
+<script>
```
Comment (gappc) [inline databrowser/src/pages/LegalPage.vue:7]: I would set the `lang="ts"` info to improve tooling support (e.g. by VSCode `volar` extension).

At the moment, `volar` shows me this message:

```
Virtual script not found, may missing <script lang="ts"> / "allowJs": true / jsconfig.json.volar
```

File: databrowser/src/pages/PrivacyPage.vue [hunk 6]
```diff
@@ -1,3 +1,14 @@
 <template>
-  <div>Privacy Page</div>
+  <AppLayout>
+    <ContentArea>Privacy Page</ContentArea>
+  </AppLayout>
 </template>
+<script>
```
Comment (gappc) [inline databrowser/src/pages/PrivacyPage.vue:6]: I would set the `lang="ts"` info to improve tooling support (e.g. by VSCode `volar` extension).

At the moment, `volar` shows me this message:

```
Virtual script not found, may missing <script lang="ts"> / "allowJs": true / jsconfig.json.volar
```

File: databrowser/src/pages/SitemapPage.vue [hunk 7]
```diff
@@ -1,3 +1,15 @@
 <template>
-  <div>Sitemap Page</div>
+  <AppLayout>
+    <ContentArea>Sitemap Page</ContentArea>
+  </AppLayout>
 </template>
+
+<script>
```
Comment (gappc) [inline databrowser/src/pages/SitemapPage.vue:7]: I would set the `lang="ts"` info to improve tooling support (e.g. by VSCode `volar` extension).

At the moment, `volar` shows me this message:

```
Virtual script not found, may missing <script lang="ts"> / "allowJs": true / jsconfig.json.volar
```

File: databrowser/src/pages/datasets/DetailViewPage.vue [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+          </TabPanel>
+          <TabPanel>Details</TabPanel>
+          <TabPanel>Raw</TabPanel>
+        </TabPanels>
+      </TabGroup>
+    </ContentArea>
+  </AppLayout>
+</template>
+
+<script class="ts">
```
Comment (gappc) [inline databrowser/src/pages/datasets/DetailViewPage.vue:30]: If think there is a typo, instead of `<script class="ts">` it should be `<script lang="ts">`

**Review by gappc — APPROVED** [review]

**Pull request merged**